src/render/GameMenuScreen.java

```diff
@@ -0,0 +1,38 @@
+package traingame.render;
+
+import java.util.*;
+
+import traingame.engine.render.gui.*;
+import traingame.engine.render.text.BitmapFont;
+import traingame.Game;
+
+public class GameMenuScreen extends Screen {
+    public GameMenuScreen(Game game) {
+        BitmapFont font = Fonts.getButtonFont();
+        selectable.add(new TextButton(font, "Resume", () -> game.escape()));
+        selectable.add(new TextButton(font, "Quit", () -> game.exitWorld()));
```

Comment:
Can you please change the following?
        selectable.add(new TextButton(font, "Quit", () -> game.exitWorld()));
to
        selectable.add(new TextButton(font, "Quit to Main Menu", () -> game.exitWorld()));

Reply:
I think that looks awkwardly long, but sure. I'll update it.